src/app/core/views/DefaultPropInfo.ts

```diff
@@ -32,3 +34,8 @@ export const DefaultWirePropInfo: PropInfo<AnyWire> = [
 ];
 
 export const DefaultPortPropInfo: PropInfo<AnyPort> = [];
+
+export const ConstantNumberPropInfo: PropInfo<ConstantNumber> = [
+    ...DefaultComponentPropInfo,
+    { id: "inputNum", type: "int", key: "inputNum", label: "Input Number", min: 0, max: 15, step: 1 },
+];
```

Comment:
Keep this in `src/app/digital/views/info/index.ts`, this file is only for PropInfo's that are shared among multiple objects (but ConstantNumber will only be used for the ConstantNumber)